Core/clim-basic/extended-streams/stream-input.lisp

```diff
@@ -666,15 +663,6 @@ known gestures."
   (with-lock-held ((state-lock pointer))
     (slot-value pointer 'button-state)))
 
-(defmethod pointer-modifier-state ((pointer standard-pointer))
-  (with-lock-held ((state-lock pointer))
-    (slot-value pointer 'modifier-state)))
-
-(defmethod pointer-update-state
-    ((pointer standard-pointer) (event keyboard-event))
-  (with-lock-held ((state-lock pointer))
-    (setf (slot-value pointer 'modifier-state) (event-modifier-state event))))
```

Comment:
I can tell from this commit, that the pointer-modifier-state could change by means of pointer-update-state (compare to the previous commit where you remove window-modifier-state).

Reply:
Yes but `pointer-update-state` is never called so `pointer-modifier-state` never changed. The modifier state is already returned by `port-modifier-state` that works and its defined in Franz manual, so I remove `pointer-modifier-state` that seems a not used, not working and not defined in spec. replica of  `port-modifier-state`. 

 The only place where `pointer-modifier-state` was used was `window-modifier-state` that I removed (instead of modify it with the using of `port-modifier-state`) for the reason written in the previous comment.